### PR TITLE
Use lib directive

### PR DESCRIPTION
**Pull request by stephendonner**
@viorelaioia / @davehunt r?

This fixes https://github.com/mozilla/mozillians-tests/issues/349 and here's the ad-hoc run: https://fx-test-jenkins-dev.stage.mozaws.net:8443/job/mozillians.adhoc/149/console

### CODE REVIEW

**Comment by viorelaioia** [comment]
Looks good, r+

**Comment by stephendonner** [comment]
More background on this (for anyone who cares, but specifically @davehunt too):

1) straight from :abayer: "the new [libraries {}] syntax is a convenience specifically for Declarative, while @Library is across all of Pipeline."
2) both will work, as we're using them (but this is preferred, hence the PR)
3) minimum required plugin + version to support the new syntax is Pipeline Model Definition[0], version 1.1.6 or greater

[0] the new syntax is a convenience specifically for Declarative, while @Library is across all of Pipeline.

**Comment by stephendonner** [comment]
@abayer - welcoming any addendums to what I've said :-)

**Comment by abayer** [comment]
Sounds right to me. =)

File: Jenkinsfile [hunk 1]
```diff
@@ -9,6 +7,9 @@ def capabilities = [
 
 pipeline {
   agent any
+  libraries {
+      lib('fxtest@1.6')
```
Comment (davehunt) [inline Jenkinsfile]: nit: match indentation of two spaces as used throughout.

**Comment by davehunt** [comment]
Oh, and we'll need to make sure we update the plugin on all Jenkins instances before we merge this.

File: Jenkinsfile (hunk 1 above)
Comment (stephendonner) [inline Jenkinsfile]: Thanks; fixed!

**Comment by davehunt** [comment]
Do not merge until [bug 1387387](https://bugzilla.mozilla.org/show_bug.cgi?id=1387387) is resolved. Thanks!

**Pull request merged**